cpp/src/parquet/types.h

```diff
@@ -479,6 +479,9 @@ struct Encoding {
   };
 };
 
+// Exposed data encodings.
+enum class ExposedEncodingType { NO_ENCODING = 0, DICTIONARY = 1 };
```

Comment:
Let's drop type from the name.  

Instead fo ExposedEncoding what do you think about ReadEncoding?

Reply:
thanks, will change it